examples/model_compress/fpgm_tf_mnist.py

```diff
@@ -0,0 +1,130 @@
+from nni.compression.tensorflow import FPGMPruner
```

Comment:
is it easy to reproduce one of the experiments in the paper?

Reply:
The key metrics of the pruning are reduced FLOPs and accuracy drop, to reproduce the reduced FLOPs, we need to use the pruned compact model, which is not implemented yet.

Reply:
Sure, I have added verification code in fpgm_torch_mnist.py  example to verify the pruned conv kernel weight sparsity. By checking the sparsity and loss, we can verify:
1.  the configured layers are pruned.
2.  the pruned model has similar loss with origin model
But this code still can not verify the implementation is same as the paper.  I am considering to add some kind of verification in UT.